test/e2e/scalability/load.go

```diff
@@ -184,6 +188,8 @@ var _ = SIGDescribe("Load capacity", func() {
 		itArg.services = os.Getenv("CREATE_SERVICES") != "false"
 
 		It(name, func() {
+			creationPhase := testPhaseDurations.StartPhase(100, "creation")
```

Comment:
Please improve the phase msgs here and below (similar to my comments for density test).

Reply:
Done.